Sources/Sentry/SentryProfiler.mm

```diff
@@ -136,7 +136,7 @@
     }
 
     const auto sample = [[SentrySample alloc] init];
-    sample.absoluteTimestamp = backtrace.absoluteTimestamp;
+    sample.absoluteTimestamp = SentryCurrentDate.getCurrentDateProvider.systemTime;
```

Comment:
m: Is this correct? Before you had a fixed time extracted from the backtrace and now you're getting the current time.

Reply:
This is a good point. this would push the timestamps gathered for backtrace samples a bit later than when they were previously gathered. When I changed it I assumed it wouldn't be a big deal, but let's check with @indragiek. Good catch @brustolin!